src/renderers/dom/shared/CSSPropertyOperations.js

```diff
@@ -125,9 +125,10 @@ var CSSPropertyOperations = {
    * The result should be HTML-escaped before insertion into the DOM.
    *
    * @param {object} styles
+   * @param {ReactDOMComponent} context
    * @return {?string}
    */
-  createMarkupForStyles: function(styles) {
+  createMarkupForStyles: function(styles, context) {
```

Comment:
Could you use the name "component" instead? "Context" often means something else (http://facebook.github.io/react/docs/context.html).